libs/vkd3d/resource.c

```diff
@@ -2819,6 +2828,18 @@ static HRESULT d3d12_resource_create(struct d3d12_device *device, uint32_t flags
             FIXME_ONCE("ReadFromSubresource may be buggy on host-visible images with ALLOW_SIMULTANEOUS_ACCESS | ALLOW_UNORDERED_ACCESS.\n");
     }
 
+    if ((flags & VKD3D_RESOURCE_COMMITTED) &&
+        device->device_info.memory_priority_features.memoryPriority &&
+        (desc->Flags & high_priority_resource_flags))
+    {
+        size_t resource_size = d3d12_resource_is_texture(object) ?
+            vkd3d_compute_resource_layouts_from_desc(device, &object->desc, NULL) :
+            object->desc.Width;
+        uint32_t adjust = vkd3d_get_priority_adjust(resource_size);
+
+        object->priority.d3d12priority = D3D12_RESIDENCY_PRIORITY_HIGH | adjust;
```

Comment:
I don't see where HEAP_FLAG_NOT_RESIDENT is handled for committed resources.

Reply:
Thought I double-checked but yes!  Fixed now.